sstable/reader.go

```diff
@@ -460,8 +462,16 @@ func (r *Reader) readBlock(
 	// Call IsTracingEnabled to avoid the allocations of boxing integers into an
 	// interface{}, unless necessary.
 	if readDuration >= slowReadTracingThreshold && r.logger.IsTracingEnabled(ctx) {
-		r.logger.Eventf(ctx, "reading %d bytes took %s",
-			int(bh.Length+block.TrailerLen), readDuration.String())
+		_, file1, line1, _ := runtime.Caller(1)
+		_, file2, line2, _ := runtime.Caller(2)
+		r.logger.Eventf(ctx, "reading block for %s of %d bytes took %s (%s/%s:%d -> %s/%s:%d)",
```

Comment:
nit: Do we have any convention for printing filenums with `.sst` at the end? At first glance it's hard to know what that int is.